backend/src/index.js

```diff
@@ -104,7 +104,7 @@ io.on( 'connection', socket => {
 			return;
 		}
 
-		issueUsers.delete( socket.session.login );
+		delete issueUsers[ socket.session.login ];
```

Comment:
We need to call `storeUsers` after it.

Reply:
I've searched this bug for one hour... 🤦‍♂